rcl/test/rcl/test_validate_enclave_name.cpp

```diff
@@ -47,6 +53,57 @@ TEST(TestValidateEnclaveName, test_validate) {
     RCL_RET_OK,
     rcl_validate_enclave_name("/foo/bar", &validation_result, &invalid_index));
   EXPECT_EQ(RCL_ENCLAVE_NAME_VALID, validation_result);
+
+  {
+    auto mock = mocking_utils::patch(
```

Comment:
Please add a comment describing that the `rmw` can return "TOO_LONG` and still be a valid case, otherwise the tests on its own looks odd later with the returned `RCL_ENCLAVE_NAME_VALID`

Reply:
Sounds good. See 39ef45a.